### DISCUSSION TITLE
Using (python) plugins on Windows

### DISCUSSION THREAD

**Discussion by quazardous** [zim-desktop-wiki/zim-desktop-wiki#2010] [Q&A]
How can I create an environment to enable plugins with python dependencies on Windows with the default compiled windows release ?

**Comment by introt** [comment]
As far as I know, they should be picked up if they're in your `PYTHONPATH`. You can try this out by eg. putting [this template](https://github.com/introt/zim-plugin-template/) in your plugin directory, making sure it loads alright, and then adding an import statement of any of the installed Python packages at the top, and see if the plugin still loads (indicating that the import was successful).

You might find these answers helpful: https://stackoverflow.com/questions/3701646/how-to-add-to-the-pythonpath-in-windows-so-it-finds-my-modules-packages
